Resources/Prototypes/Entities/Objects/Specific/Mech/mechs.yml

```diff
@@ -92,6 +92,19 @@
   - type: GuideHelp
     guides:
     - Robotics
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 400
+      behaviors:
```

Comment:
Is this similar to the value in 13?

Reply:
Ripley already have 200 integrity hp as in 13, but getting 0 integrity hp doesn't remove mech from the map, but make it in "broken state", even if get 10000 amount of damage when in "broken state", mech's never getting completely removed from the map. This code adds that when mech get 400 amount of damage it will completely removed from the map (and all other mechs that instantiates from BaseMech)

Reply:
Didn't find on how much damage mechs are destroys completely on 13